pkg/sdkserver/sdkserver.go

```diff
@@ -587,3 +604,68 @@ func (s *SDKServer) healthy() bool {
 	defer s.healthMutex.RUnlock()
 	return s.healthFailureCount < s.health.FailureThreshold
 }
+
+/* Alpha Functionality */
+
+// PlayerConnect should be called when a player connects.
+func (s *SDKServer) PlayerConnect(ctx context.Context, id *alpha.PlayerId) (*alpha.Empty, error) {
+	panic("implement me")
+}
+
+// PlayerDisconnect should be called when a player disconnects.
+func (s *SDKServer) PlayerDisconnect(ctx context.Context, id *alpha.PlayerId) (*alpha.Empty, error) {
+	panic("implement me")
+}
+
+// GetPlayerCount returns the current player count.
+func (s *SDKServer) GetPlayerCount(ctx context.Context, _ *alpha.Empty) (*alpha.Count, error) {
+	panic("implement me")
+}
+
+// SetPlayerCapacity to change the game server's player capacity.
+func (s *SDKServer) SetPlayerCapacity(ctx context.Context, count *alpha.Count) (*alpha.Empty, error) {
+	if !runtime.FeatureEnabled(runtime.FeaturePlayerTracking) {
+		return nil, errors.New(string(runtime.FeaturePlayerTracking) + " not enabled")
+	}
+	s.gsUpdateMutex.Lock()
+	s.gsPlayerCapacity = count.Count
+	s.gsUpdateMutex.Unlock()
+	s.workerqueue.Enqueue(cache.ExplicitKey(string(updatePlayerCapacity)))
+
+	return &alpha.Empty{}, nil
+}
+
+// GetPlayerCapacity returns the current player capacity.
+func (s *SDKServer) GetPlayerCapacity(ctx context.Context, _ *alpha.Empty) (*alpha.Count, error) {
+	if !runtime.FeatureEnabled(runtime.FeaturePlayerTracking) {
+		return nil, errors.New(string(runtime.FeaturePlayerTracking) + " not enabled")
+	}
+	s.gsUpdateMutex.RLock()
+	defer s.gsUpdateMutex.RUnlock()
+
+	count := &alpha.Count{
```

Comment:
nit: It seems like this could just be inlined:

`return &alpha.Count{Count: s.gsPlayerCapacity}, nil`

Reply:
You still left this on two lines. Was there a reason to keep the local `count` variable that I'm missing? It seems unnecessary.

Reply:
Ah, I missed a bit. Fixed!